polaris.shopify.com/content/components/link/index.md

```diff
@@ -49,6 +49,37 @@ The HTML that renders for the `Button` and `Link` components carries meaning. Us
 - a more cohesive visual experience for sighted users
 - products that are easier to maintain at scale
 
+### Open a new tab only when necessary
+
+Use default links whenever possible to avoid disorienting merchants and causing accessibility problems by opening a new tab.
+
+Links should be indicated as external when merchants are:
+
+- Performing a task or workflow, like creating a product
+- Navigating to a page outside of the Shopify admin
+
+### No icon
+
+Avoid using the [external link icon](/icons?icon=ExternalMinor&q=exter), as it can add unnecessary visual load inside a sentence or when accompanied by other content. Instead, add clarity to external links through clear link text and predictable placement of the link in a merchant’s workflow.
+
+Edge case: External icons should not be used as an indicator that a new tab or window will be opened. However, they may be used sparingly in features where symbols help merchants scan and pick from a list of several kinds of navigation options, like the admin's global search results.
```

Comment:
Should this be “External **link** icons”?

Reply:
Good question. The icon name is "external" so that's why we'd went with that but they are just used for external links right now. The thing that is a little confusing is that the "Promote" icon looks an awful like the external icon 🤔 
http://localhost:3000/icons?icon=ExternalSmallMinor&q=external